<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Untitled[*] - Automatic Calendar and Event Management</string>
  </property>
  <property name="iconSize">
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>780</width>
         <height>507</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSchedule">
    <property name="title">
     <string>&amp;Schedule</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionOptimize"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_Task">
    <property name="title">
     <string>&amp;Task</string>
    </property>
    <addaction name="actionAdd"/>
   </widget>
   <addaction name="menuSchedule"/>
   <addaction name="menu_Task"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionAdd"/>
   <addaction name="separator"/>
   <addaction name="actionOptimize"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad">
   <property name="text">
    <string>&amp;Load</string>
   </property>
   <property name="toolTip">
    <string>Load Schedule</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save Schedule</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>&amp;Add</string>
   </property>
   <property name="toolTip">
    <string>Add Task</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOptimize">
   <property name="text">
    <string>&amp;Optimize</string>
   </property>
   <property name="toolTip">
    <string>Optimize Schedule</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="toolTip">
    <string>About AutoCal</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit AutoCal</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
